apps/src/responsive.jsx

```diff
@@ -0,0 +1,119 @@
+/* A very simple responsive layout system.
+ */
+
+import * as utils from './utils';
+import $ from 'jquery';
+
+export default class Responsive {
+
+  constructor(responsiveWindowWidth) {
+    this.responsiveWindowWidth = responsiveWindowWidth;
+  }
+
+  /**
+   * Gets the container width.
+   * Returns either a number (e.g. 1170) or a string (e.g. "97%").
```

Comment:
comment should be 970?

Reply:
There are constants somewhere that Brad made that set 970 as our page width. Let's use those.